src/llvm-late-gc-lowering.cpp

```diff
@@ -781,6 +781,9 @@ void LateLowerGCFrame::LiftPhi(State &S, PHINode *Phi) {
                             assert(isa<Instruction>(BaseElem) && "Unknown value type detected!");
                             InsertBefore = cast<Instruction>(BaseElem)->getNextNonDebugInstruction();
```

Comment:
LLVM will soon have `getInsertionPointAfterDef`, which would be an even more correct implementation for this PR to use here